pkg/volume/csi/csi_block.go

```diff
@@ -191,7 +191,7 @@ func (m *csiBlockMapper) stageVolumeForBlock(
 		return "", err
 	}
 
-	klog.V(4).Infof(log("blockMapper.stageVolumeForBlock successfully requested NodeStageVolume [%s]", stagingPath))
+	klog.V(4).InfoS(log("BlockMapper.stageVolumeForBlock successfully requested NodeStageVolume"))
```

Comment:
The argument for `stagingPath` has been dropped here

Reply:
There is already a log(at line 175) that contains the `stagingPath`, I think it is not necessary to print it again.